FormFrameworkREADME.md

```diff
@@ -12,6 +13,28 @@ One of the benefits of the Form Framework tool is that it allows simple coupling
 * [Back-End keywords glossary](#back-end-keywords-glossary)
 ----
 
+## Setup for Local development
+Install the following to run the application locally
+  1. Node 10+
+  2. Docker
+  3. Postgres
+  4. GNU make (for windows)
+
+After installation, follow the instructions below to start local development
+  1. To install all the dependencies, go to the following folder from the root directory and run either `npm install` or `npm ci`
+      - `api/api_sources` - Installs the packages required by the backend node application
+      - `app/lucy` - To install the packages needed for the angular application
+  2. Navigate to `api` directory and run either of the following
+      - `make local-debug` - To run the application with logs
+      - `make local` - To start the application in the background and hide the logs
```

Comment:
Add entry for 
- make api: Shell into the api container. Local api container is customized with all development related CLI tools. Using api container shell is recommended.